### PR TITLE
Code formatting.

### PR DESCRIPTION
**Pull request by splatteredbits**
Let's get all our code to follow current standards in one big change. This should make reviewing pull requests easier since we will no longer need to distinguish what changes are important vs. formatting.

* Parameter names should be prefixed with their types (don't put parameter types on their own line).
* Use terse set-attribute-boolean-property-to-true syntax (i.e. `[Parameter(Mandatory)]` not `[Parameter(Mandatory=$true)])`.
* Whiskey's context is strongly typed so replace parameter types where it is typed as an `[object]` with `[Whiskey.Context]`.

### CODE REVIEW

File: Whiskey/Tasks/Pipeline.ps1 [hunk 1]
```diff
@@ -1,15 +1,13 @@
  function Invoke-WhiskeyPipelineTask
 {
     [CmdletBinding()]
-    [Whiskey.Task("Pipeline", SupportsClean=$true, SupportsInitialize=$true)]
+    [Whiskey.Task("Pipeline", SupportsClean, SupportsInitialize)]
```
Comment (DecoyJoe) [inline Whiskey/Tasks/Pipeline.ps1]: Single quotes around `Pipeline` and remove spaces after the commas.

File: Whiskey/Tasks/PowerShell.ps1 [hunk 2]
```diff
@@ -1,16 +1,14 @@
 
 function Invoke-WhiskeyPowerShell
 {
-    [Whiskey.Task("PowerShell",SupportsClean=$true,SupportsInitialize=$true)]
+    [Whiskey.Task("PowerShell",SupportsClean,SupportsInitialize)]
```
Comment (DecoyJoe) [inline Whiskey/Tasks/PowerShell.ps1]: Single quotes around `PowerShell`

File: Whiskey/Tasks/ProGetUniversalPackage.ps1 [hunk 3]
```diff
@@ -129,12 +127,10 @@ function New-WhiskeyProGetUniversalPackage
...
         param(
-            [Parameter(Mandatory=$true)]
-            [object[]]
-            $Path,
+            [Parameter(Mandatory)]
+            [object[]]$Path,
 
-            [Switch]
-            $AsThirdPartyItem
+            [Switch]$AsThirdPartyItem
```
Comment (DecoyJoe) [inline Whiskey/Tasks/ProGetUniversalPackage.ps1]: Lowercase `[switch]`

File: Whiskey/Tasks/SetVariable.ps1 [hunk 4]
```diff
@@ -2,15 +2,13 @@
 function Set-WhiskeyVariable 
 {
     [CmdletBinding()]
-    [Whiskey.Task("SetVariable",SupportsClean=$true,SupportsInitialize=$true)]
+    [Whiskey.Task("SetVariable",SupportsClean,SupportsInitialize)]
```
Comment (DecoyJoe) [inline Whiskey/Tasks/SetVariable.ps1]: Single quotes around `SetVariable`

File: Whiskey/Tasks/PublishProGetUniversalPackage.ps1 [hunk 5]
```diff
@@ -5,13 +5,11 @@ function Publish-WhiskeyProGetUniversalPackage
     [Whiskey.Task("PublishProGetUniversalPackage")]
```
Comment (DecoyJoe) [inline Whiskey/Tasks/PublishProGetUniversalPackage.ps1]: Single quotes around `PublishProGetUniversalPackage`

File: Whiskey/Tasks/TaskDefaults.ps1 [hunk 6]
```diff
@@ -2,15 +2,13 @@
 function Set-WhiskeyTaskDefaults
 {
     [CmdletBinding()]
-    [Whiskey.Task("TaskDefaults",SupportsClean=$true,SupportsInitialize=$true)]
+    [Whiskey.Task("TaskDefaults",SupportsClean,SupportsInitialize)]
```
Comment (DecoyJoe) [inline Whiskey/Tasks/TaskDefaults.ps1]: Single quotes around `TaskDefaults`

File: Whiskey/Tasks/Version.ps1 [hunk 7]
```diff
@@ -5,16 +5,13 @@
     [Whiskey.Task("Version")]
```
Comment (DecoyJoe) [inline Whiskey/Tasks/Version.ps1]: Single quotes around `Version`

File: build.ps1 [hunk 8]
```diff
@@ -26,20 +26,16 @@ Starts a build and uses "Release" as the build configuration when building the W
 [CmdletBinding(DefaultParameterSetName='Build')]
 param(
     [Parameter(Mandatory=$true,ParameterSetName='Clean')]
-    [Switch]
     # Runs the build in clean mode, which removes any files, tools, packages created by previous builds.
-    $Clean,
+    [Switch]$Clean,
```
Comment (DecoyJoe) [inline build.ps1]: Lowercase `[switch]`

File: build.ps1 [hunk 9]
```diff
@@ -26,20 +26,16 @@ Starts a build and uses "Release" as the build configuration when building the W
...
-    [Switch]
     # Runs the build in clean mode, which removes any files, tools, packages created by previous builds.
-    $Clean,
+    [Switch]$Clean,
 
     [Parameter(Mandatory=$true,ParameterSetName='Initialize')]
-    [Switch]
     # Initializes the repository.
-    $Initialize,
+    [Switch]$Initialize,
```
Comment (DecoyJoe) [inline build.ps1]: Lowercase `[switch]`

File: Whiskey/Functions/Get-WhiskeyTasks.ps1 [hunk 10]
```diff
@@ -21,9 +21,8 @@ function Get-WhiskeyTask
     [CmdLetBinding()]
     [OutputType([Whiskey.TaskAttribute])]
     param(
-        [Switch]
         # Return tasks that are obsolete. Otherwise, no obsolete tasks are returned.
-        $Force
+        [Switch]$Force
```
Comment (DecoyJoe) [inline Whiskey/Functions/Get-WhiskeyTasks.ps1]: Lowercase `[switch]`

File: Whiskey/Functions/Install-WhiskeyNode.ps1 [hunk 11]
```diff
@@ -4,17 +4,14 @@ function Install-WhiskeyNode
...
         [Parameter(Mandatory)]
-        [string]
         # The directory where Node should be installed. Will actually be installed into `Join-Path -Path $InstallRoot -ChildPath '.node'`.
-        $InstallRoot,
+        [string]$InstallRoot,
 
-        [Switch]
         # Are we running in clean mode? If so, don't re-install the tool.
-        $InCleanMode,
+        [Switch]$InCleanMode,
```
Comment (DecoyJoe) [inline Whiskey/Functions/Install-WhiskeyNode.ps1]: Lowercase `[switch]`

File: Whiskey/Functions/Install-WhiskeyNodeModule.ps1 [hunk 12]
```diff
@@ -23,31 +23,25 @@ function Install-WhiskeyNodeModule
...
         [Parameter(Mandatory)]
-        [string]
         # The path to the build root.
-        $BuildRootPath,
+        [string]$BuildRootPath,
 
-        [Switch]
         # Whether or not to install the module globally.
-        $Global,
+        [Switch]$Global,
```
Comment (DecoyJoe) [inline Whiskey/Functions/Install-WhiskeyNodeModule.ps1]: Lowercase `[switch]`

File: Whiskey/Functions/Install-WhiskeyNodeModule.ps1 [hunk 13]
```diff
@@ -23,31 +23,25 @@ function Install-WhiskeyNodeModule
...
 
-        [Switch]
         # Are we running in clean mode?
-        $InCleanMode
+        [Switch]$InCleanMode
```
Comment (DecoyJoe) [inline Whiskey/Functions/Install-WhiskeyNodeModule.ps1]: Lowercase `[switch]`

File: Whiskey/Functions/Invoke-WhiskeyBuild.ps1 [hunk 14]
```diff
@@ -54,26 +54,22 @@ function Invoke-WhiskeyBuild
...
         # If you pass a value to this parameter, the `Publish` pipeline is *not* run implicitly. You must pass its name to run it.
-        $PipelineName,
+        [string[]]$PipelineName,
 
-        [Parameter(Mandatory=$true,ParameterSetName='Clean')]
-        [Switch]
+        [Parameter(Mandatory,ParameterSetName='Clean')]
         # Runs the build in clean mode. In clean mode, tasks delete any artifacts they create, including downloaded tools and dependencies. This is opt-in, so if a task is not deleting its artifacts, it needs to be updated to support clean mode.
-        $Clean,
+        [Switch]$Clean,
```
Comment (DecoyJoe) [inline Whiskey/Functions/Invoke-WhiskeyBuild.ps1]: Lowercase `[switch]`

File: Whiskey/Functions/Invoke-WhiskeyBuild.ps1 [hunk 15]
```diff
@@ -54,26 +54,22 @@ function Invoke-WhiskeyBuild
...
 
-        [Parameter(Mandatory=$true,ParameterSetName='Initialize')]
-        [Switch]
+        [Parameter(Mandatory,ParameterSetName='Initialize')]
         # Runs the build in initialize mode. In initialize mode, tasks download/install/configure any tools/dependencies they use/need during the build. Initialize mode is intended to be used by developers so that any tools/dependencies they need can be downloaded/installe/configured without needing to run an entire build, which can sometimes take a long time.
-        $Initialize
+        [Switch]$Initialize
```
Comment (DecoyJoe) [inline Whiskey/Functions/Invoke-WhiskeyBuild.ps1]: Lowercase `[switch]`

File: Whiskey/Tasks/ProGetUniversalPackage.ps1 [hunk 16]
```diff
@@ -129,12 +127,10 @@ function New-WhiskeyProGetUniversalPackage
     function Copy-ToPackage
     {
         param(
-            [Parameter(Mandatory=$true)]
-            [object[]]
-            $Path,
+            [Parameter(Mandatory)]
+            [object[]]$Path,
```
Comment (DecoyJoe) [inline Whiskey/Tasks/ProGetUniversalPackage.ps1]: Should use the same case as the fully qualified type name `[System.Object]` as no `[object]` type accelerator exists:
```
❯ ([PSObject].Assembly.GetType('System.Management.Automation.TypeAccelerators')::Get).GetEnumerator() | Where-Object { $_.Value.Name -eq 'Object' }

❯
```

**Comment by splatteredbits** [comment]
@DecoyJoe OK. Ready for next review.

**Pull request merged**